cel/canonical_eventlog_test.go

```diff
@@ -72,23 +85,62 @@ func TestCELMeasureAndReplay(t *testing.T) {
 	}
 
 	cel := &CEL{}
+	celRTMR := &CEL{}
 
 	cosEvent := CosTlv{ImageRefType, []byte("docker.io/bazel/experimental/test:latest")}
+	digestMap, err := GenerateDigestMap(measuredHashes, cosEvent)
+	if err != nil {
+		t.Fatal(err)
+	}
+	digestMapRTMR, err := GenerateDigestMap([]crypto.Hash{crypto.SHA384}, cosEvent)
+	if err != nil {
+		t.Fatal(err)
+	}
+
 	someEvent2 := make([]byte, 10)
 	rand.Read(someEvent2)
 	cosEvent2 := CosTlv{ImageDigestType, someEvent2}
-	appendOrFatal(t, cel, tpm, test.DebugPCR, measuredHashes, cosEvent)
-	appendOrFatal(t, cel, tpm, test.DebugPCR, measuredHashes, cosEvent2)
+	digestMap2, err := GenerateDigestMap(measuredHashes, cosEvent2)
+	if err != nil {
+		t.Fatal(err)
+	}
+	digestMapRTMR2, err := GenerateDigestMap([]crypto.Hash{crypto.SHA384}, cosEvent2)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	appendOrFatal(t, cel, test.DebugPCR, cosEvent, digestMap)
+	extendTpmPcr(t, tpm, test.DebugPCR, digestMap)
+	appendOrFatal(t, celRTMR, CosEventLogRegisterIndex, cosEvent, digestMapRTMR)
+	rtmr.ExtendDigestClient(fakeRTMR, CosRTMR, digestMapRTMR[crypto.SHA384])
+
+	appendOrFatal(t, cel, test.DebugPCR, cosEvent2, digestMap2)
+	extendTpmPcr(t, tpm, test.DebugPCR, digestMap2)
```

Comment:
nit: make an extendPCRAndLogOrFatal instead of calling extendTpmPcr and appendOrFatal